tox.ini

```diff
@@ -8,14 +8,16 @@ deps =
     jsonschema
     pytest
     pyyaml
+    pytest-cov
```

Comment:
It's nice to keep the dependencies in order:
```suggestion
    pytest-cov
    pyyaml
```

Reply:
Done, thank you!